diff -up ./lib/gcstar/GCModels/GCcomics.gcm.orig ./lib/gcstar/GCModels/GCcomics.gcm
--- gcstar/lib/gcstar/GCModels/GCcomics.gcm.orig	2016-07-25 01:32:48.026796517 +0200
+++ gcstar/lib/gcstar/GCModels/GCcomics.gcm	2016-07-25 01:33:24.256584154 +0200
@@ -79,7 +79,7 @@ name="GCcomics">
 				label="Volume"
 				init="1"		
 				min="0"
-				max="9999"
+				max="1000000"
 				group="main"
 				imported="true"/>				
 		<field value="title"
